.travis.yml

```diff
@@ -44,9 +42,6 @@ before_deploy:
   - docker --version
   - pip install --user awscli # install aws cli w/o sudo
   - export PATH=$PATH:$HOME/.local/bin # put aws in the path
-  - mv e2e/REPORTS dist/REPORTS # Add reports to dist
```

Comment:
what about the REPORTS

Reply:
Why do we need those in the final image?